x/auth/keeper/bech32_codec.go

```diff
@@ -21,6 +24,10 @@ func NewBech32Codec(prefix string) address.Codec {
 
 // StringToBytes encodes text to bytes
 func (bc bech32Codec) StringToBytes(text string) ([]byte, error) {
```

Comment:
I didn't notice they didn't exist in auth, but imo we should have tests for these.
Btw, why is it duplicated in codec and in auth?

Reply:
well we want to avoid everything depending on auth if it doesnt alreay depend on it. I put it in testutil first cause it was meant to be for testing then it was suggested to put it in codec/address. Id prefer to keep it only for testing

Reply:
Alright, then I think we should still keep the implementation in sync. We should add these changes in `codec/address` as well.
But, auth is actually using the `codec/address`: https://github.com/cosmos/cosmos-sdk/pull/15347/files#diff-0ed9fdcfb33c700c91a7fbc70a765cab16244144aed4b504f190ce5d657dbc96R210, so it isn't used for testing only.

Reply:
oh weird then thats an over sight. Ill amend it here